blog-comments-simple/Makefile

```diff
@@ -0,0 +1,10 @@
+
+readme:
```

Comment:
I don't want to remove the other files.
I think they are useful for someone playing around.